File: JavaScript/8js 4/stringy.js
/*
|--------------------------------------------------------------------------
| Stringy Strings
|--------------------------------------------------------------------------
|
*/

function stringy(size) {
   res = "";
   for (let i = 1; i <= size; i++) {
      res += i % 2 == 0 ? "0" : "1";
   }
   return res;
}


function stringy(size) {
   return "".padStart(size, "10");
}


function stringy(size) {
   return "10".repeat(size).slice(0, size);
}


console.log(stringy(4), "1010");

File: Python/8py 2/stringy.py
"""
|--------------------------------------------------------------------------
| Stringy Strings
|--------------------------------------------------------------------------
|
"""

def stringy(size):
   return "".join([str(i%2) for i in range(1,size+1)])


def stringy(size):
   return ('10' * size)[:size]


def stringy(size):
   return '10'*(size//2)+'1'*(size%2)


print(stringy(4), "1010")